specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Diagnostics.json

```diff
@@ -432,6 +432,154 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/detectors": {
```

Comment:
Would you please unify the resource name? In the API path, it's called "detector", but in the model and examples, it's called "JobDiagnostic"

Reply:
The "detector" is used across many resources as a kind of sub-type. Let me use Jobs_ListDetectors and so on.

Reply:
Updated